package.json

```diff
@@ -41,8 +41,9 @@
     "test:unit:watch": "cross-env NODE_ENV=test babel-node test/watch.js unit"
   },
   "peerDependencies": {
-    "react": "^15.4.0",
-    "react-dom": "^15.4.0",
+    "react": "^15.5.0",
+    "react-dom": "^15.5.0",
+    "react-prop-types": "^15.5.0",
```

Comment:
1) It should be a dependency.
2) The package is called `prop-types`, **not** `react-prop-types`.